src/Common/Commands.ScenarioTest/Resources/Automation/AutomationTests.ps1

```diff
@@ -17,7 +17,8 @@
 Checks whether the first string contains the second one
 #>
 
-$accountName='AutomationAccount'
+$accountName='safeer'
```

Comment:
nit: it's little bit awkward to have account name with a real person. Maybe `automatedAccount` is better?`